dist/images/Dockerfile.fedora

```diff
@@ -22,7 +22,7 @@ ARG TARGETOS
 ARG TARGETARCH
 
 WORKDIR /workspace
-RUN apt-get update && apt-get install -y -qq make git
+RUN dnf install --best --refresh -y make git
```

Comment:
do we need to do dnf update first? (ah refresh is present so ok)

Reply:
Yep, `--refresh` is equivalent to `dnf upgrade`